frontend/src/screens/researcher/researcher-profile.tsx

```diff
@@ -0,0 +1,217 @@
+    const [researcher, setResearcher] = useState(useCurrentResearcher())
+    if (!researcher) {
+        return null
+    }
+    const imageURL = researcher.avatarUrl || DefaultAvatar;
+    const AvatarImage = styled.img({
+        borderRadius: '50%',
+        border: `1px solid ${colors.lightGray}`,
+        padding: researcher.avatarUrl ? 0 : 25,
+        height: 125,
+        width: 125,
+    })
+    const [avatar, setAvatar] = useState<Blob>()
+    const [isShowingModal, setShowingModal] = useState(false)
+    const onHide = () => setShowingModal(false)
+
+    const saveResearcher = async (researcher: Researcher) => {
+        if (!researcher.id) {
+            return;
+        }
+        const r = await api.updateResearcherAvatar({
+            id: researcher.id,
+            avatar,
+        })
+        setResearcher(r)
+        onHide()
+    }
+
+
+    return (
+        <Box className='col-2' justify='start' direction='column'>
+            <Box onClick={() => setShowingModal(true)} direction='column' align='center' gap='large' css={{ cursor: 'pointer' }} >
+                <AvatarImage alt="User Avatar" src={imageURL}/>
+
+                <Box align='baseline' gap>
+                    <a className='links'>{researcher.avatarUrl ? 'Change Image' : 'Upload Image'}</a>
+                    <Tooltip tooltip='Upload a picture that best introduces you to learners'>
+                        <Icon css={{ color: colors.tooltipBlue }} icon='questionCircleFill' height={14}/>
+                    </Tooltip>
+                </Box>
+            </Box>
+
+            <Modal
+                onHide={onHide}
+                center
+                show={isShowingModal}
+                large
+                data-test-id="update-avatar-modal"
+                title='Update Avatar'
+            >
+                <Modal.Body>
+                    <Form
+                        onSubmit={saveResearcher}
+                        showControls
+                        onCancel={onHide}
+                        defaultValues={researcher}
+                    >
+                        <Box direction='column' align='center' justify='center' gap>
+                            <FileUploader name='avatar' onChange={setAvatar} accept='image/jpg, image/png' />
+                        </Box>
+                    </Form>
+                </Modal.Body>
+            </Modal>
+        </Box>
+    )
+}
+
+const PageWrapper = styled(Box)({
+    backgroundColor: colors.pageBackground,
+    flexDirection: 'column',
+    height: '100vh',
+})
+
+const ProfileSection = styled(Box)({
+    backgroundColor: colors.white,
+    border: `1px solid ${colors.lightGray}`,
+    borderRadius: 5,
+    padding: 30,
+})
+
+const Resources = styled(Box)({
+    maxHeight: 250,
+    border: `1px solid ${colors.lightGray}`,
+    marginTop: 40,
+    borderRadius: 5,
+    backgroundColor: colors.white,
+    padding: 20,
+    'a': {
+        color: colors.grayText,
+    },
+})
```

Comment:
we should remember these and export them if we need something similar in the future